src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,24 @@
+import { useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+import { Tabs } from '../components/Tabs';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams<{ tabId: string }>();
+  const activeTab = tabs.find(tab => tab.id === tabId);
```

Comment:
make sense to use useMemo here, because it's important to performance

Reply:
Done